spot_wrapper/wrapper.py

```diff
@@ -411,63 +411,61 @@ def wrapper_try_claim(self, *args, **kwargs):
     return wrapper_try_claim
 
 
+@dataclass()
```

Comment:
should this just be `@dataclass` rather than `@dataclass()` - i haven't seen the latter syntax before